packages/flutter_test/lib/src/window.dart

```diff
@@ -97,6 +104,12 @@ class TestWindow implements ui.SingletonFlutterWindow {
     onMetricsChanged?.call();
   }
 
+  /// Just [physicalSizeTestValue], but automatically reset after the test finishes
+  set physicalSizeTestValueAutoClear(ui.Size value) { // ignore: avoid_setters_without_getters
```

Comment:
Add a doc reference to this new auto clear method in the docs for `physicalSizeTestValue`.